docs/tutorials/get-started/blink-an-led.md

```diff
@@ -0,0 +1,416 @@
+For example, "GPIO 22".
+**When working with the GPIO pins with Viam, you will use the board pin numbers.**
+For example, board pin 38 is the same pin as GPIO pin 20, but when configuring your robot with Viam, you should refer to it as pin `38`.
+
+Some of the GPIO pins can be used not only as GPIO pins but for other more specific functions as well.
+The website [pinout.xyz](https://pinout.xyz/) is a helpful resource with the exact layout and role of each pin.
+
+### Wire your circuit
+
+{{% alert title="Caution" color="caution" %}}
+Disconnect your Raspberry Pi and any other electronics from power before touching or moving wires on them.
+{{% /alert %}}
+
+Now it's time to wire your circuit according to the diagram below.
+It doesn't matter what color jumper wires you use but for reference, the blue wire below connects the LED to ground, and the red wire connects the resistor to pin 8 (GPIO 14).
+
+<img src="../../img/blink-an-led/circuit.png" alt="Circuit diagram showing a Raspberry Pi with a red connector running out of GPIO pin 8 to a 100-ohm resistor. The resistor is connected to the long lead of a red LED bulb. Finally, a blue connector connects the short lead of the LED to the ground connection on pin 6 of the Raspberry Pi GPIO pins." width="100%">
+
+The resistor and LED need to be in series as in the diagram above.
+To find the right resistor use the resistor color code -- for a 100 ohm resistor, it needs to be brown-black-brown.
+You can use your multimeter to double-check the resistor value or check yours using the photo below.
+
+<img src="../../img/blink-an-led/resistor.jpg" alt="Photo of a 100-ohm resistor with text overlaid that says, in order, brown-black-brown-gold." width="50%">
+
+When hooking up the circuit, **note the *polarity* of the LED**.
+You will notice that the LED has long and short leads.
+The long lead is the positive side, which is known as the anode.
+The short lead is the negative side, which is known as the cathode.
+Connect the long anode to the resistor and the short cathode to ground (pin 6) on the Raspberry Pi.
+
+## Configure your robot
+
+Before proceeding, be sure that you have [connected your Pi to the Viam app](/installation/#install-viam-server).
+
+Now it's time to configure your robot's components.
+First, go to the [Viam app](https://app.viam.com/) on your web browser and navigate to your robot's **Config** tab.
+
+<img src="../../img/blink-an-led/create-component.png" alt="Screenshot of the Viam app showing the Create component field on the Components subtab of the Config tab." width="100%">
+
+You will add a [*board component*](/components/board/) to represent your single board computer, which in this case is the Raspberry Pi.
+You'll create the new component in the **Create component** panel.
+
+- Name the board whatever you like as long as you are consistent when referring to it later; we'll name it `local` since it is the board we will communicate with directly.
```

Comment:
Do you mind adding tabs here with the config in raw json on the second tab? like on the guardian tutorial for example?

Reply:
What is the objective of that? If we strongly want people to use the config builder, why add extra to an absolute beginner tutorial?

Reply:
I think it's nice to show the alternatives for people who don't want to use the builder UI and I would like us to add that to all tutorials. Then it would be the odd one out.

Reply:
ok